.github/workflows/release-go-cmd-osrm-files-extractor.yml

```diff
@@ -0,0 +1,34 @@
+name: Release cmd/osrm-files-extractor
```

Comment:
If I want to release `OASIS` or other tools, I think I also need to create similar yml file.

Reply:
You can just copy this file to "release-go-cmd-oasis.yml", then rename the `name`, `CMD_PATH ` and `BINARY_NAME` , that's all. No need to modify anything else.